core/src/verification/queue/kind.rs

```diff
@@ -24,6 +23,7 @@ pub use self::headers::Headers;
 use crate::consensus::CodeChainEngine;
 use crate::error::Error;
 use crate::service::ClientIoMessage;
+use primitives::U256;
```

Comment:
Please remove the blank line after `use rlp::*;` following the CodeChain-io/codechain#1940.